HtmlCssReview/HtmlCssReviewCodeAlong.md

```diff
@@ -0,0 +1,136 @@
+# HTML & CSS Review Code-Along
+Start by forking [this empty Repl](https://replit.com/@HylandOutreach/EmptyWeb), then follow the instructions below to build a simple webpage!
+
+To fork the Repl, click the "Fork Repl" button on the page:
+
+![](../Assets/ForkRepl.png)
+
+## Add the Basic HTML Elements
+Add the foundational elements to the **index.html** file. Start with `<html></html>`, then add `<head></head>` and `<body></body>` between the `html` tags.
+
+```html
+<html>
+    <head>
+    </head>
+    <body>
+    </body>
+</html>
+```
+
+## Add a Header
+Between the opening and closing `body` tags, add an `h3` element. Within the `h3` element, add the proper text.
+
+```html
+<h3>Fortnite Fan Site</h3>
+```
+
+## Add an Image
+Add an image to the page.
+
+1. Go to [Google Images](https://google.com/images/)
+1. Find an image, and copy the image address/URL/location
+1. Create an `img` element, and set its `src` attribute to be the URL for the image
+
+```html
+<img src="https://assets.reedpopcdn.com/Fortnite-Battle-Pass-creen-1.jpeg" />
+```
+
+## Add a Paragraph
+Under the `img` element, add a `p` element that contains the text for the paragraph.
+
+```html
+<p>Fortnite is a great video game filled with several memorable characters and lots of action.</p>
+```
+
+## Add Another Header
+Under the `p`, add an `h3` header that will serve as the title for a list.
+
+```html
+<h3>My Top Three Zones</h3>
+```
+
+## Add a List
```

Comment:
I don't think we actually covered lists in the HTML/CSS trimester - we can probably just use paragraphs for this for simplicity